src/components/Dashboard.tsx

```diff
@@ -0,0 +1,10 @@
+import { DashboardAppBar } from './DashboardAppBar'
+
+export const Dashboard = () => {
+  return (
+    <div>
+      <DashboardAppBar />
+      <h1>Dashboard</h1>
+    </div>
```

Comment:
Do you need that `div` wrapper? It looks like react fragments should be enough.
```suggestion
    <>
      <DashboardAppBar />
      <h1>Dashboard</h1>
    </>
```

Reply:
All unnecessary divs will be changed to react fragments